tianshou/__init__.py

```diff
@@ -9,4 +9,5 @@
     "policy",
     "trainer",
     "exploration",
+    "highlevel",
```

Comment:
highlevel is not imported here, so this change doesn't affect functionality. It's not clear to me what caused the issue you refer to

Reply:
The issue is typically caused by someone not using the right version of tianshou.
This import is not required.